sheets-import/README.md

```diff
@@ -1,102 +1,270 @@
-This template requires app-specific configuration before it can used.
-Specifically, the template will need to be informed of the authorization
-details, and certain adjustments made to ensure the correct data can be
-extracted from the responses.
-
-In the Server.gs, APICode.gs and Auth.gs project files, there are several
-comments marked as **TODO**. To configure the template, visit each of these
-**TODO** sections and follow the directions found there.
-
-Note that the template design assumes that the template will connect to an API
-service using OAuth 2.0; however, it would be possible adapt it to connect to
-a different kind of service, such as a SQL database.
-
-## Additional information
-
-For more information, see:
-
-* [Extending Google Sheets](https://developers.google.com/apps-script/guides/sheets)
-* [Known Issues specific to Google Sheets](https://developers.google.com/apps-script/migration/sheets)
-
-Note that this template must be added to a container-bound
-script attached to a Google Sheet in order to function. Developed
-add-ons must go through a
-[publishing process](https://developers.google.com/apps-script/add-ons/publish)
-before they can be made available publicly.
+# sheets-import
+
+An Add-on Template for importing data from a third-party source into
+Google Sheets.
+
+**Features:**
+
+- Rapid integration with third-party data sources like APIs or internal
+  databases,
+- Configurable options for each report;
+  **add text fields, checkboxes, and selectboxes** and reference them in
+  your API call,
+- Scheduled reports that run every 24 hours,
+- Collaborative reports shared between users,
+- Update sheets in the background.
+
+![A demo of sheets-import integrated with GitHub](media/demo.gif)
+
+## Quickstart
+
+#### 1. Create the Apps Script Project
+
+1. Navigate to [https://script.google.com](https://script.google.com) and create
+  a new `Blank Project`.
+2. For each file in this repo, create a file of the
+  correct type and paste in the contents.
+  - For an HTML file, select `File` > `New` > `Html File` and enter the
+    full name (i.e. `app.js.html`).
+  - For a Script file, select `File` > `New` > `Script File` and enter the
+    full name (i.e. `server.gs`).
+  - The files you must copy over are:
+    - [app.js.html](/raw/master/app.js.html),
+    - [authorize.html](/raw/master/authorize.html),
+    - [config.gs](/raw/master/config.gs),
+    - [index.html](/raw/master/index.html),
+    - [main.html](/raw/master/main.html),
+    - [myService.gs](/raw/master/myService.gs),
+    - [myServiceAuthCallbackView.html](/raw/master/myServiceAuthCallbackView.html),
+    - [myServiceReauthorizationEmail.html](/raw/master/myServiceReauthorizationEmail.html),
+    - [report.html](/raw/master/report.html),
+    - [util.gs](/raw/master/util.gs).
+
+#### 2. Add Required Libraries to the Project
+
+  1. Click `Resources` > `Libraries`.
+  2. Paste `MswhXl8fVhTFUH_Q3UOJbXvxhMjh3Sh48` (Underscore) into the
```

Comment:
If it's possible, see if you can eliminate the need for the Underscore library from the template. It may make the code a little more complex, but will remove a setup step.